src/__tests__/common/helper.ts

```diff
@@ -11,18 +11,26 @@ export const getSubmitButton = (): HTMLElement => {
 export const getSubmitButtonProps = (): Record<string, TFrontendEngineFieldSchema> => {
 	return {
 		[SUBMIT_BUTTON_ID]: {
-			label: "Submit",
+			label: SUBMIT_BUTTON_LABEL,
 			fieldType: "submit",
 		},
 	};
 };
 
-export const getField = (role: TAriaRoles, label?: string, isQuery = false): HTMLElement => {
+export const getField = (role: TAriaRoles, config?: string | ByRoleOptions, isQuery = false): HTMLElement => {
```

Comment:
instead of `config`, can we have a more descriptive name? because it is now a name and a config